src/jobs/run_codacy_plugins_test.yml

```diff
@@ -0,0 +1,36 @@
+description: "Runs codacy-plugins-test on a tool"
+
+executor: machine
+
+parameters:
+  codacy_plugins_test_version:
+    type: string
+    description: "codacy-plugins-test version"
+
+environment:
+  CODACY_PLUGINS_TEST_VERSION: << parameters.codacy_plugins_test_version >>
```

Comment:
I would say that if the environment variable is not used anywhere else, use `<< parameters.codacy_plugins_test_version >>`